src/lib.rs

```diff
@@ -62,17 +70,24 @@ pub type Result<T, E = CoreError> = std::result::Result<T, E>;
 /// expected that only one instance of an implementation will exist for the lifetime of the
 /// worker(s) using it.
 pub trait Core: Send + Sync {
-    /// Ask the core for some work, returning a [Task], which will eventually contain either a
-    /// [protos::coresdk::WfActivation] or an [protos::coresdk::ActivityTask]. It is then the
-    /// language SDK's responsibility to call the appropriate code with the provided inputs.
+    /// Ask the core for some work, returning a [Task], which will contain a [protos::coresdk::WfActivation].
+    /// It is then the language SDK's responsibility to call the appropriate code with the provided inputs.
     ///
     /// TODO: Examples
+    /// TODO: rename to poll_workflow_task and change result type to WfActivation
```

Comment:
Might as well do the rename here

Reply:
Yes, I thought about doing that, but didn't want to bloat the scope of this change. Also making result type WfActivation would require moving task token from Task to WfActivation level.